tests/lean/run/doc_string6.lean

```diff
@@ -0,0 +1,10 @@
+/--
+/- /- /- nested docstring -/ -/ -/
```

Comment:
Wow, it was already failing for regular comments?

Reply:
Yeah, opening this in 3.7.2c gives the following:
```
doc string of `foo` was:
/ // // nested docstring -- -- -/

expected:
/- /- /- nested docstring -/ -/ -/
```